packages/commonwealth/client/scripts/views/components/NewThreadForm/NewThreadForm.tsx

```diff
@@ -101,6 +117,18 @@ export const NewThreadForm = () => {
     return threadTitle || getTextFromDelta(threadContentDelta).length > 0;
   }, [threadContentDelta, threadTitle]);
 
+  const showGatingBanner = useMemo(() => {
```

Comment:
we can remove this condition and just directly use `restrictedTopicIds.includes(threadTopic.id)` with some state var for closing the banner